source/auth/auth.rst

```diff
@@ -863,18 +860,27 @@ Body                     Action=GetCallerIdentity&Version=2011-06-15\\n
         ``*``, Denotes a header that MUST be included in SignedHeaders, if present.
 
 .. note::
-        Region is not a header, but simply part of the authorization header. Region by default is ‘us-east-1’ since this is the 
-        implicit region for ‘sts.amazonaws.com’. Drivers will need to derive the region to use from the endpoint. The region 
-        is the second piece of a FQDN name. While all official AWS STS endpoints start with “sts.”, there are non-AWS hosted 
-        endpoints and test endpoints that will not follow this rule.
-
-================= =========
-Host              Region
-================= =========
-sts.amazonaws.com us-east-1
-first.second      second
-first             us-east-1
-================= ========= 
+        Region is not a header, but simply part of the authorization header. The general syntax of a regional endpoint (or host) is as follows.
+
+.. code:: javascript
+
+	service-code.region-code.amazonaws.com
+|
+Drivers MUST derive the region (the ``region-code`` above) from the second label of the host. If a region is not present, or does not contain a full stop "``.``" character (*dot* or *period*) as a seperator, then the default region, ``us-east-1`` MUST be used. Drivers MUST error on any empty labels or if the host itself is empty or longer than 255 bytes. Examples are provided below. 
```

Comment:
Suggest adding back the "longer than 255 bytes per RFC 1035" link so it's obviously where the limitation comes from.

Reply:
Done